src/effects/Glitch.tsx

```diff
@@ -5,7 +5,7 @@ import { useVector2 } from '../util'
 
 export type GlitchProps = ConstructorParameters<typeof GlitchEffect>[0] &
   Partial<{
-    mode: typeof GlitchMode[keyof typeof GlitchMode]
+    mode: (typeof GlitchMode)[keyof typeof GlitchMode]
```

Comment:
For complex user-facing types, I'd try to export a type alias -- `export type GlitchMode = (typeof _GlitchMode)[keyof typeof _GlitchMode]`.